.eslintrc.js

```diff
@@ -23,13 +32,10 @@ module.exports = {
       }
     ],
     '@typescript-eslint/await-thenable': 'off',
-    '@typescript-eslint/ban-ts-ignore': 'error',
```

Comment:
Why disable this rule? I think `@ts-ignore` should remain banned in favor of [`@ts-expect-error`](https://devblogs.microsoft.com/typescript/announcing-typescript-3-9/#ts-expect-error-comments)

Reply:
I think it was removed in newest versions (gonna check)

Reply:
yea, it is removed in newest version